lib/puppet/provider/cs_clone/pcs.rb

```diff
@@ -17,6 +17,7 @@
 
   defaultfor operatingsystem: [:fedora, :centos, :redhat]
 
+  # TODO : discuss of change_clone_id (conflict between pcs behavior and request in issue#149)
```

Comment:
We need to discuss this.
Commit for issue #149 uses cibadmin instead of pcs to modify the cluster configuration:  Red Hat does NOT support clusters configured with commands other than pcs.

We should at least ask the feedback from pcs developers about the change of the default clone-id '<primitive>-clone' in the cib. It might lead to unexpected side effects in the future if developers rely on this specific clone-id format.

Reply:
@roidelapluie for review

Reply:
will you take that action? The dev of pcs is @cfeist IIRC

Reply:
I believe the current maintainers are @idevat and @tomjelinek. I will ask on #clusterlabs.

Reply:
Feedback from ClusterLabs dev: It should work just fine. However, to be 100% sure it would be needed to look through code or write tests.